js/novadax.js

```diff
@@ -99,35 +101,35 @@ module.exports = class novadax extends Exchange {
             },
             'api': {
                 'public': {
-                    'get': [
-                        'common/symbol',
-                        'common/symbols',
-                        'common/timestamp',
-                        'market/tickers',
-                        'market/ticker',
-                        'market/depth',
-                        'market/trades',
-                        'market/kline/history',
-                    ],
+                    'get': {
+                        'common/symbol': 1.2,
+                        'common/symbols': 1.2,
+                        'common/timestamp': 1.2,
+                        'market/tickers': 1.2,
+                        'market/ticker': 1.2,
+                        'market/depth': 1.2,
+                        'market/trades': 1.2,
+                        'market/kline/history': 1.2,
+                    },
                 },
                 'private': {
-                    'get': [
-                        'orders/get',
-                        'orders/list',
-                        'orders/fill',
-                        'orders/fills',
-                        'account/getBalance',
-                        'account/subs',
-                        'account/subs/balance',
-                        'account/subs/transfer/record',
-                        'wallet/query/deposit-withdraw',
-                    ],
-                    'post': [
-                        'orders/create',
-                        'orders/cancel',
-                        'account/withdraw/coin',
-                        'account/subs/transfer',
-                    ],
+                    'get': {
+                        'orders/get': 2,
```

Comment:
These values contradict the comment above, the comment says private endpoints cost 1.2, however, we set 1.2 for public endpoints and then set 2 for the private ones. I thought we would set a rate-limiting cost of 1 for the cheapest public endpoint.

The novadax API says: 

> Public endpoints: up to 60 requests per second for each IP;
> Private endpoints: up to 20 requests per second for each API AccessKey.

So, i guess, the cost must be 1 (public) and 3 (private) ?

Reply:
you are correct, my bad I must have confused it with docs of another one of the exchanges I was working on whilst awaiting verifications etc. 

(and its 1.2 is the only weight that works for the public api, throws ratelimit errors when the weight is 1 as it should be as stated in the docs)